integration/integration_test_utils.py

```diff
@@ -68,40 +73,101 @@ def __init__(self, **kwargs):
             'X-v3io-session-key': self._access_key
         }
 
-
 def append_return(lst, x):
     lst.append(x)
     return lst
 
-
 def _generate_table_name(prefix='bigdata/storey_ci/Aggr_test'):
     random_table = ''.join([random.choice(string.ascii_letters) for i in range(10)])
-    return f'{prefix}/{random_table}'
+    return f'{prefix}/{random_table}/'
 
+redis_server = None
 
-@pytest.fixture()
-def setup_teardown_test():
+def get_redis_client(redis_fake_server=None):
+    REDIS_URL = os.environ.get('REDIS_URL')
+    if REDIS_URL:
+        return r.Redis.from_url(REDIS_URL)
+    else:
+        return fakeredis.FakeRedis(decode_responses=True, server = redis_fake_server)
+
+def redis_driver(redis_fake_server = None, *args, **kwargs):
+    return RedisDriver(redis_client = get_redis_client(redis_fake_server),key_prefix="storey-test:", *args, **kwargs)
+
+def get_driver(setup_teardown_test, *args, **kwargs):
+    if setup_teardown_test["driver_name"] == "V3ioDriver":
+        return V3ioDriver(*args, **kwargs)
+    elif setup_teardown_test["driver_name"] == "RedisDriver":
+        redis_fake_server = setup_teardown_test["redis_fake_server"] if "redis_fake_server" in setup_teardown_test else None
+        return redis_driver(redis_fake_server = redis_fake_server, *args, **kwargs)
+    else:
+        assert 0
+
+def remove_redis_table(table_name):
+    redis_client = get_redis_client()
+    count = 0
+    ns_keys = "storey-test:" + table_name + "*"
+    for key in redis_client.scan_iter(ns_keys):
+        redis_client.delete(key)
+        count += 1
+
+drivers_list = ["V3ioDriver", "RedisDriver"]
+@pytest.fixture(params=drivers_list)
+def setup_teardown_test(request):
     # Setup
     table_name = _generate_table_name()
+    
+    test_params = {}
+    driver_name = request.param
+    test_params["driver_name"] = driver_name
+    test_params["table_name"] = table_name
+    if driver_name == "RedisDriver":
+        REDIS_URL = os.environ.get('REDIS_URL')
+        if not REDIS_URL:
+            test_params["redis_fake_server"] = fakeredis.FakeServer()
```

Comment:
I'm starting to think this code is repeated a lot... Can we extract it?
```suggestion
        redis_url = os.environ.get('REDIS_URL')
        if not redis_url:
            test_params["redis_fake_server"] = fakeredis.FakeServer()
```

Reply:
done